client/src/store/authReducer.js

```diff
@@ -0,0 +1,95 @@
+import Cookies from "js-cookie";
+
+const LOGIN = "auth/login";
+const CREATE = "auth/create";
+const LOGOUT = "auth/logout";
+
+
+const authReducer = (state = {}, action) => {
+    switch(action.type) {
+        case LOGIN:
+            return {id: action.id};
+        case CREATE:
+            return {id: action.id};
```

Comment:
`LOGIN` and `CREATE` actions can be combined since they do the same thing! You should also add the `username` for displaying that throughout the app!